status/flowcell.py

```diff
@@ -142,8 +142,14 @@ def fetch_ont_run_stats(view_all, view_project, run_name):
 
         run_dict["experiment_name"], run_dict["sample_name"], name = run_dict["TACA_run_path"].split("/")
         
-        run_date, run_dict["start_time"], run_dict["position"], run_dict["flow_cell_id"], run_dict["run_id"] = name.split("_")
-        run_dict["start_date"] = datetime.datetime.strptime(str(run_date), '%Y%m%d').strftime('%Y-%m-%d')
+        run_date, run_dict["start_time"], instr_or_pos, run_dict["flow_cell_id"], run_dict["run_id"] = name.split("_")
```

Comment:
Why can't this split be done in the view where the run name is already split from the path?
`run_name = last(doc['run_path'].split("/"))`

Then we don't need to have a separate case for ongoing here, just one to add additional fields if the run is finished.

Reply:
It could, but for an ongoing run all of the data wrangling is done in the back-end either way.

Right now, the view for an ongoing run returns ONLY the objects that were put into the document by TACA (run path and run status). The run name is only split from the path out of necessity for getting the appropriate key for the view.

It's true we could further extrapolate data in the view instead of the back-end, but I'd prefer not to separate the data wrangling over two separate files / languages / locations.